src/main/xar-resources/data/advanced-installation/advanced-installation.xml

```diff
@@ -185,4 +188,43 @@
             </varlistentry>
         </variablelist>
     </sect1>
+    <sect1 xml:id="cli-flags">
+        <title>Command line flags to set host, ports and configuration</title>
+        <para>There are several java tool options available to change the host, http port
```

Comment:
http ~~port~~ and

Reply:
@line-o Am I understanding correctly from your review here that there are still planned changes on this PR? I'll hold off on merging until you confirm it's ready.

Reply:
I'll do that now. So that we can get this merged.